part_1/src/com/freeit/vpohodaiev/lesson2/homework_task1.java

```diff
@@ -1,9 +1,16 @@
 package com.freeit.vpohodaiev.lesson2;
 
+import java.util.Scanner;
+
 public class homework_task1 {
     public static void main (String [] args) {
-        int x = 5;
-        int y = 8;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Fill in x number and click on Enter");
+        int x;
+        x = sc.nextInt();
+        System.out.println("Fill in y number and click on Enter");
+        int y ;
+        y = sc.nextInt();
         if (x<y)
```

Comment:
According to Java Code Convention math operators should be separated with whitespaces from operands.

Reply:
Just use formatting for this (Ctrl + Alt + L)